doc/api/child_process.md

```diff
@@ -325,6 +325,7 @@ changes:
 * `args` {string[]} List of string arguments.
 * `options` {Object}
   * `cwd` {string} Current working directory of the child process.
+  * `detached` {boolean} If `true`, Prepare child to run independently of its parent
```

Comment:
Thank you!
Doc nits:
```suggestion
  * `detached` {boolean} If `true`, prepares child to run independently of its parent.
```

Reply:
Will change it! Thanks for letting me know.

Also, ne za chto (Noticed your name)